distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -33,6 +33,7 @@
         <exclude>etc/org.apache.karaf.log.cfg</exclude>
         <exclude>etc/org.ops4j.pax.web.cfg</exclude>
         <exclude>etc/users.properties</exclude>
+        <exclude>etc/startup.properties</exclude>
```

Comment:
I think you should leave a comment starting that this should be reverted once karaf does move to log4j2